packages/wrangler/src/utils/log-file.ts

```diff
@@ -62,21 +62,30 @@ ${message}
 	if (!hasLoggedLocation) {
 		hasLoggedLocation = true;
 		const relativeFilepath = path.relative(process.cwd(), debugLogFilepath);
-		logger.debug(`🐛 Writing logs to "${relativeFilepath}"`); // use logger.debug here to not show this message by default -- since logging to a file is no longer opt-in
+		logger.debug(`🪵 Writing logs to "${relativeFilepath}"`); // use logger.debug here to not show this message by default -- since logging to a file is no longer opt-in
 		onExit(() => {
-			console.info(`🐛 Logs were written to "${relativeFilepath}"`);
+			// only print the log file location if the log file contains an error message
+			// TODO(consider): recommend opening an issue with the contents of this file?
+			if (hasSeenErrorMessage) {
+				logger.warn(`🪵 Logs were written to "${relativeFilepath}"`); // use logger.warn here so not to pollute stdout -- some commands print json to stdout
+			}
 		});
 	}
 
+	if (!hasSeenErrorMessage) {
+		// TODO(consider): adding `|| messageLevel === "warn"`
+		hasSeenErrorMessage = messageLevel === "error";
+	}
+
```

Comment:
Does this create multiple `onExit` hooks? Do they need to be cleaned up?

Reply:
The onExit call is guarded by an if-block ensuring it's only called once